.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+    ignore:
+      # ignore forked linters because of their versioning issues. TODO: fix it.
+      - dependency-name: "github.com/golangci/*"
+  - package-ecosystem: github-actions
```

Comment:
so great to have this one to update github-actions as well.

I just take a look at the all supported package-ecosystem. Docker might be another one we can enable.

Reply:
let me check it

Reply:
done